ovn_cluster.sh

```diff
@@ -446,6 +446,39 @@ function start-db-cluster() {
     $SSL_ARGS start_northd
 }
 
+function check_etcd_mode() {
```

Comment:
NIT: How about we rename this function to nb_etcd or something other than "check"?

Reply:
is_ovsdb_etcd_mode() any better ?

Reply:
> is_ovsdb_etcd_mode() any better ?

yes!